src/components/tables/table-children/TableChildBody.tsx

```diff
@@ -21,27 +22,45 @@ export interface ITableBodyInheritedFromTableProps {
     withoutHoverOnRow?: boolean;
 }
 
+export type StateRow = 'error' | 'warning' | 'success';
+
 export interface ITableChildBodyProps extends ITableBodyInheritedFromTableProps {
     tableId: string;
     rowData: IData;
     isLoading: boolean;
     onRowClick?: (actions: IActionOptions[]) => void;
+    disabled?: boolean;
     isMultiSelect: boolean;
     withoutHoverOnRow?: boolean;
+    state?: StateRow;
     handleOnRowClick?: (actions: IActionOptions[], rowData: IData) => void;
 }
 
+const getStatusRowStyle = (state: StateRow, content: JSXRenderable) => {
+    const svgStyle: string = classNames([

```

Comment:
Rename const for `svgClasses` because we're dealing with classes here, not to be confused with the real `style` prop